simulate.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+
+import numpy as np
+import scipy as sp
+from scipy import signal
+import h5py
+import math
+import PSS_utils as utils
+import sys
+sys.path.append('/Users/nearclouding/Desktop/UTGRV_REU/nanograv_group')
+import VersionZeroPointZero as PSS
```

Comment:
This won't find VersionZeroPointZero, instead you need to call the files you need like so:

`from . import PSS_utils as utils`

Also, you shouldn't need the `import sys` or the line after it, since everything will be living in the same folder on anyones computers.